src/media/js/views/search.js

```diff
@@ -208,9 +208,16 @@ define('views/search',
             if (query) break;
         }
 
-        if (query && queryParam === 'author') {
-            pageTypes += ' leaf';
-            utilsLocal.headerTitle(gettext('Developer Listing'));
+        if (query) {
+            switch (queryParam) {
+                case 'author':
+                    pageTypes += ' leaf';
```

Comment:
I did like this - 
`var pageTypes = ['search app-list', 'leaf'];
..........
..........
        if (query) {
            switch (queryParam) {
                case 'author':
                    pageTypes.join(' ');
                    utilsLocal.headerTitle(gettext('Developer Listing'));
                    break;
                case 'q':
                    pageTypes.join(' ');
                    utilsLocal.headerTitle(title);
            }
        } else if (!settings.meowEnabled) {
            pageTypes.join(' ');
        }
`